<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerCalculusModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerCalculusModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,2,0">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Stone Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,5">
      <item>
       <widget class="qSlicerCalculusFooBarWidget" name="FooBar" native="true"/>
      </item>
      <item>
       <widget class="QFrame" name="comboxFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>input volume</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="inputVolumeMRMLNodeComboBox" native="true">
           <property name="nodeTypes" stdset="0">
            <string>vtkMRMLScalarVolumeNode</string>
           </property>
           <property name="noneEnabled" stdset="0">
            <bool>true</bool>
           </property>
           <property name="addEnabled" stdset="0">
            <bool>false</bool>
           </property>
           <property name="removeEnabled" stdset="0">
            <bool>true</bool>
           </property>
           <property name="renameEnabled" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="tableblock_2">
           <column>
            <property name="text">
             <string>MAX</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>MIN</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>AVERAGE</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>AOD</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>IOD</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="6">
             <widget class="QPushButton" name="acqStoneBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>任意角度采集</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="sliceThickSpinBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="suffix">
               <string>mm</string>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>20.000000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="6">
             <widget class="QPushButton" name="y_verticalAcqStoneBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>垂直Y轴采集</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>38</height>
               </size>
              </property>
              <property name="text">
               <string>u water：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>38</height>
               </size>
              </property>
              <property name="text">
               <string>slice thicknesss:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="6">
             <widget class="QPushButton" name="z_verticalAcqStoneBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>垂直Z轴采集</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="openBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>开启</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="uWaterSpinBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="value">
               <double>0.183000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QDoubleSpinBox" name="materialThickSpinBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>20.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="6">
             <widget class="QPushButton" name="x_verticalAcqStoneBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>垂直X轴采集</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>38</height>
               </size>
              </property>
              <property name="text">
               <string>material thickness:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QPushButton" name="clearButton_2">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QPushButton" name="saveButton_2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>save</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton_2" native="true">
     <property name="text" stdset="0">
      <string>Reformat Area</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
      <item>
       <widget class="qSlicerCalculusReformatWidget" name="reformatWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton_3" native="true">
     <property name="text" stdset="0">
      <string>Bed Hole Parameter</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>markups</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="markupsMRMLNodeComboBox" native="true">
        <property name="nodeTypes" stdset="0">
         <string>vtkMRMLMarkupsFiducialNode</string>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>input volume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="inputVolumeMRMLNodeComboBox_2" native="true">
        <property name="nodeTypes" stdset="0">
         <string>vtkMRMLScalarVolumeNode</string>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTableWidget" name="tableblock">
        <column>
         <property name="text">
          <string>MAX</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MIN</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>AVERAGE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>AOD</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IOD</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>采集床孔数据</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
   <slots>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerCalculusFooBarWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerCalculusFooBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerCalculusReformatWidget</class>
   <extends>QWidget</extends>
   <header>qslicercalculusreformatwidget.h</header>
   <container>1</container>
   <slots>
    <slot>onMRMLSceneChanged(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerCalculusModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputVolumeMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>3</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCalculusModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>reformatWidget</receiver>
   <slot>onMRMLSceneChanged(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCalculusModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupsMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCalculusModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputVolumeMRMLNodeComboBox_2</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
 </slots>
</ui>
